gnomad_qc/v3/sample_qc/subpop_analysis.py

```diff
@@ -388,6 +392,7 @@ def main(args):  # noqa: D103
                 withhold_prop=args.withhold_prop,
                 pop=pop,
                 curated_subpops=CURATED_SUBPOPS[pop],
+                additional_samples_to_drop=outliers_ht,
```

Comment:
This will only be set if `args.run_subpop_pca` is True too right?

Reply:
ah good point, I can move the assignment of outliers_ht up so it won't be dependent on any of the run args